lib/utils/app_config.dart

```diff
@@ -1,8 +1,11 @@
-import 'dart:io';
+import 'package:flutter/foundation.dart';
 
 /// Use `if (mobileDevice)` when you want different behavior
 /// based on whether a touchscreen or mouse cursor is being used.
-final mobileDevice = switch (Platform.operatingSystem) { 'ios' || 'android' => true, _ => false };
+final mobileDevice = switch (defaultTargetPlatform) {
+  TargetPlatform.android || TargetPlatform.iOS => true,
+  _ => false,
+};
```

Comment:
Completely unrelated, but while going through the original PR, I realized that Flutter web apps don't have access to `Platform.operatingSystem`, so you gotta use `defaultTargetPlatform` instead.